Automattic/wp-calypso
# Update the domain connection support URLs

#### Proposed Changes

* This PR updates two of the domain connection support URLs

#### Testing Instructions

Connect a domain and go to the start of the connection management flow.

Ex. domains/mapping/a8ctest.com/setup/a8c-tests.blog?step=suggested_update

Make sure that the "support documentation" link goes to https://wordpress.com/support/domains/connect-existing-domain/#step-2-change-your-domains-name-servers

![Connect_a_Domain_Setup_‹_A8C_Test_Site_—_WordPress_com](https://user-images.githubusercontent.com/1379730/197014886-df46f328-20bb-44d8-8884-628056f69184.jpg)

Also visit the advanced connection flow and make sure that the "Support documentation" links to https://wordpress.com/support/domains/connect-existing-domain/#step-2-add-the-dns-records

![Cursor_and_Connect_a_Domain_Setup_‹_A8C_Test_Site_—_WordPress_com](https://user-images.githubusercontent.com/1379730/197015551-cb022076-47a1-4f5d-90be-d6c1a6a19faf.jpg)


Related to #68548
